src/blant.c

```diff
@@ -1379,14 +1381,15 @@ int main(int argc, char *argv[])
 	    if(_confidence >= 1) _confidence /= 100; // user specified percent
 	    break;
 	case 'k': _k = atoi(optarg);
+		_k_base = MIN(_k, OLD_MAX_K);
 	    if (_GRAPH_GEN && _k >= 33) {
 		_k_small = _k % 10;
```

Comment:
WTF? Mod 10? Why?

Reply:
This is the first thing that came to my mind when I saw your comment 🤣🤣
![image](https://github.com/user-attachments/assets/c04604e5-12fe-46a1-a62c-351fbf422ad2)